pyproject.toml

```diff
@@ -54,32 +54,21 @@ include = ["dbt/adapters", "dbt/include", "dbt/__init__.py"]
 [tool.hatch.envs.default]
 dependencies = [
     "dbt_common @ git+https://github.com/dbt-labs/dbt-common.git",
-]
-
-[tool.hatch.envs.lint]
```

Comment:
There's no point in separate lint and typecheck environments. We want the default to be able to run precommit, which requires all of this in default.